pkg/datapath/linux/config/config.go

```diff
@@ -399,6 +403,11 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 		}
 	}
 
+	// Manually defined a tunnel endpoint IPv4 CIDR
+	if tunnelEndpointIPv4CIDR := option.Config.GetTunnelEndpointIPv4CIDR(); tunnelEndpointIPv4CIDR != nil {
+		cDefinesMap["TUNNEL_ENDPOINT_IPV4_CIDR"] = tunnelEndpointIPv4CIDR.String()
+	}
```

Comment:
I don't see any BPF code corresponding to this macro declaration. Is there another piece that we're not seeing or is this only for informational purposes?